### PR TITLE
Fix issue where multiple clients were not getting last published

### PR DESCRIPTION
**Pull request by neelam-kushwah**
This commit addresses issue #4 and #6

### CODE REVIEW

File: app/src/main/java/org/eclipse/uprotocol/simulatorproxy/BaseService.java
Comment (matthewd0123) [inline app/src/main/java/org/eclipse/uprotocol/simulatorproxy/BaseService.java:1]: Remove Commented Code on lines 97 and 101

Comment (matthewd0123) [inline app/src/main/java/org/eclipse/uprotocol/simulatorproxy/BaseService.java:1]: Add method descriptions for new methods

File: app/src/main/java/org/eclipse/uprotocol/simulatorproxy/SimulatorProxyService.java
Comment (matthewd0123) [inline app/src/main/java/org/eclipse/uprotocol/simulatorproxy/SimulatorProxyService.java:1]: Remove commented code lines 530-532

Comment (neelam-kushwah) [reply app/src/main/java/org/eclipse/uprotocol/simulatorproxy/SimulatorProxyService.java:1]: We would need this code in future. That's why I added comment also

File: app/src/main/java/org/eclipse/uprotocol/simulatorproxy/BaseService.java
Comment (neelam-kushwah) [reply app/src/main/java/org/eclipse/uprotocol/simulatorproxy/BaseService.java:1]: Done

**Review by princemac — APPROVED** [review]

**Pull request merged**